foundation-extensions/profile/README.md

```diff
@@ -158,11 +188,19 @@ public class MobileApp extends Application {
 {% endtab %}
 
 {% tab title="iOS (AEP 3.x)" %}
-### Objective C
 
-**Required**: You must complete the following steps in the app before calling other `UserProfile` APIs.
+### Swift
+
+```swift
+// AppDelegate.swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    MobileCore.registerExtensions([UserProfile.self], {
+  })
+  ...
```

Comment:
return true

Reply:
The return value should depend on the customer's code logic. I think we don't need to add it here. We show the same for other extensions:
https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension#register-edge-with-mobile-core